Sketch.java

```diff
@@ -1,36 +1,137 @@
 import processing.core.PApplet;
+import processing.core.PImage;
+import java.util.Set;
+import java.util.HashSet;
+
 
 public class Sketch extends PApplet {
 	
-	
+  Snow[] circles;
+  Player player;
+  public static final int SIZE = 400;
+  int boost = 0;
+  Set<Snow> hitSnowBalls = new HashSet<>();
```

Comment:
sooo what is a `HashSet` anyway?  Not on the list of things I want you to use for this assignment.

Reply:
The set keeps track of how many snowballs hit the blue circle, and make sure the same snow ball would not hit it twice.

Reply:
The set keeps track of how many snowballs hit the blue circle, and make sure the same snow ball would not hit it twice.